commands/new_function.go

```diff
@@ -114,6 +115,13 @@ func runNewFunction(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("got unexpected error while updating .gitignore file: %s", err)
 	}
 
+	var imageName string
+	if len(imagePrefix) > 0 {
+		imageName = imagePrefix + "/" + functionName
```

Comment:
What about `strings.Trim()` ? for `  ` whitespace

--prefix="alexellis "

Reply:
Thank you!
Will change it to this.

```
var imageName string
imagePrefix = strings.TrimSpace(imagePrefix)
if len(imagePrefix) > 0 {
	imageName = imagePrefix + "/" + functionName
} else {
	imageName = functionName
}
```